mods/ra/rules/fakes.yaml

```diff
@@ -1,4 +1,4 @@
-FPWR:
+﻿FPWR:
```

Comment:
I'm not sure what causes this

Reply:
Looks like an addition/removal of the UTF-8 byte-order mark. If you use Notepad++, make sure the bottom reports UTF-8 and not UTF-8-BOM.

![image](https://user-images.githubusercontent.com/1136302/60393635-c483c080-9b18-11e9-8439-2449361048f0.png)

Reply:
I'm using Visual Studio Community and by default it is already encoded Unicode UTF-8

Reply:
VS Community adds BOM by default.

https://stackoverflow.com/questions/5406172/utf-8-without-bom

Reply:
for now used a different editor